katalog/thanos/base/components/compactor/kustomization.yaml

```
@@ -0,0 +1,21 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: monitoring
+
+resources:
+  - serviceaccount.yaml
+  - pvc.yaml
+  - service.yaml
+  - deployment.yaml
+  - servicemonitor.yaml
+
+images:
+  - name: thanos
+    newName: registry.sighup.io/fury/thanos
```

this should have been
```suggestion
    newName: registry.sighup.io/fury/thanos/thanos
```
to follow our convention.

Maybe for the next version we can change this.

cc @nutellinoit